android/allegro_activity/src/Clipboard.java

```diff
@@ -0,0 +1,88 @@
+package org.liballeg.android;
```

Comment:
This file has mixed tabs/spaces, and since it's completely new, it's worth keeping it consistent.